CHANGELOG.md

```diff
@@ -15,6 +15,41 @@ our [guidelines for writing a good changelog entry](https://github.com/biomejs/b
 
 ### CLI
 
+#### New features
+
+- Add a new option `--rule` to the command `biome lint` ([#58](https://github.com/biomejs/biome/issues/58)).
+
+  This new option allows you to execute a single rule. This option is convenient to test a rule or apply the code fixes of a single rule.
+
+  For example, you can execute the `style/useNamingConvention` rule on a set of files:
+
+  ```shell
+  biome lint --rule=style/useNamingConvention src/main.js
```

Comment:
The reason why I suggested two files, is because the previous sentence says "set of files", hence the example shows a set of files.

I suggest reviewing the wording.